src/blocks/Header/Header.tsx

```diff
@@ -9,9 +9,9 @@ import {ThemeValueContext} from '../../context/theme/ThemeValueContext';
 import {Col, Grid, Row} from '../../grid';
 import {ClassNameProps, HeaderBlockBackground, HeaderBlockProps, WithChildren} from '../../models';
 import {block, getThemedValue} from '../../utils';
-import {getImageSize, getTitleSizes, titleWithImageSizes} from './utils';
 
 import './Header.scss';
+import {getImageSize, getTitleSizes, titleWithImageSizes} from './utils';
```

Comment:
this is also some interesting things, maybe we have to discuss it in opensorce chat?